client/cli/src/runtime.rs

```diff
@@ -121,7 +122,7 @@ where
 
 	let service = service_builder(config)?;
 
-	let informant_future = sc_informant::build(&service);
+	let informant_future = sc_informant::build(&service, colour_output);
```

Comment:
You can just put `true` here all the time. No need to add a field to `Configuration`.